### ISSUE TITLE
Error parsing line: A166616

### ISSUE THREAD

**Issue by jonmaiga** [loda-lang/loda-cpp#36]
This error occurred directly (for all processes) after mine_parallel.sh had restarted all processes:
```
2021-09-24 16:39:04|INFO |Using LODA home directory /home/jon/loda/
2021-09-24 16:39:04|ERROR|Error parsing line: A166616: 
2021-09-24 16:39:04|INFO |Starting LODA v21.9.21. See https://loda-lang.org
terminate called after throwing an instance of 'std::runtime_error'
2021-09-24 16:39:04|INFO   what():  |Using LODA home directory /home/jon/loda/
Error parsing line: A166616: 
2021-09-24 16:39:04|INFO |Starting LODA v21.9.21. See https://loda-lang.org
2021-09-24 16:39:04|INFO |Using LODA home directory /home/jon/loda/
2021-09-24 16:39:04|ERROR|Error parsing line: A166616: 
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error parsing line: A166616: 
2021-09-24 16:39:04|ERROR|Error parsing line: A166616: 
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error parsing line: A166616: 
...
```

All processes complained at A166616.

And finally:
```
2021-09-24 22:39:05|ERROR|Error parsing line: A166616: 
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error parsing line: A166616: 
./mine_parallel.sh: line 93: 2229830 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229831 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229832 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229833 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229834 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229835 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
./mine_parallel.sh: line 93: 2229836 Aborted                 (core dumped) loda mine -l ${log_level} -i ${i} $@
...
```

**Comment by ckrause** [comment]
Can you please try to delete `loda/lists/invalid_matches.txt` and restart the miners?

**Comment by jonmaiga** [comment]
After deleting the file the miners can start again.

**Comment by ckrause** [comment]
Ok, will be fixed in the next version. I hope to be able to release it next week.

**Comment by ckrause** [comment]
fixed in v21.9.28
